app/models/spree/order_decorator.rb

```diff
@@ -8,7 +8,7 @@ def self.prepended(base)
     end
 
     def finalize_with_create_subscription!
-      create_subscription_if_eligible
+      CreateSubscription.new(self).create
```

Comment:
@sherryson How about doing this with ActiveJob? That would allow us to create the subscription out of band, and should speed up the final step of the checkout.